### ISSUE TITLE
Travis CI not working

### ISSUE THREAD

**Issue by Tozturk18** [WheatonCS/Lexos#1077]
When trying to merge a branch with the master AppVeyor and Travis CI do unit tests to check if the code is working before merging it with the master. AppVeyor passes all the tests, however, Travis CI does not run and stays in limbo forever.

This issue is most probably related to .travis.yml file

**Comment by Tozturk18** [comment]
I made an account on Travis CI, and tried to fix .travis.yml according to their documentation but the Travis CI unit tests are still not working. 

It seems like the Travis CI website not even registering our merge request. It is instantly disregarding it. It gives some error messages such as:

Sender Tozturk18 is not allowed to trigger build in this repo.

and

Owner WheatonCS is not on a new pricing.

and 

We are unable to start your build at this time. You exceeded the number of users allowed for your plan. Please review your plan details and follow the steps to resolution.

We can pay Travis to move to a professional account but I would argue that we don't really need it since AppVeyor does almost the same things...

**Comment by JeffreyYStewart** [comment]
I agree. As I understand it AppVeyor and Travis do the same CI checks, but are catered toward different operating systems. Does that sound about right?
I don't know how significant the differences are and how important it is to use both.

**Comment by scottkleinman** [comment]
Archived both AppVeyeor and Travis and implemented GitHub Actions workflow. That makes this issue no longer relevant.
